apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -329,6 +333,25 @@ export default function RootStateDrawer() {
                     description="Title, subtitle, and Call-to-Action"
                     icon={TYPE_TO_ICON.hero}
                   />
+                ) : pageLayout === ISOMER_USABLE_PAGE_LAYOUTS.Database ? (
+                  <VStack gap="1rem" w="100%" align="start">
+                    <BaseBlock
+                      onClick={() => {
+                        setDrawerState({ state: "metadataEditor" })
```

Comment:
comment: feels abit iffy to have this - should we have a function called `getMetadataType`